UnitTesting-master/src/test/java/seleniumMail/Pages/SeleniumEasyBootstrapDemoPage.java

```diff
@@ -0,0 +1,42 @@
+package seleniumMail.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class SeleniumEasyBootstrapDemoPage {
+
+    private static By downloadBtn = By.cssSelector("#cricle-btn");
+    private static By loadingPercentage = By.xpath("//div[@class='percenttext']");
+
+    private static WebDriver driver;
+
+    public SeleniumEasyBootstrapDemoPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public SeleniumEasyBootstrapDemoPage download() {
+        driver.findElement(downloadBtn).click();
+        return this;
+    }
+
+    public boolean waitForDownloadLimitAndCheckLimit(int limit) {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(50));
+        wait.pollingEvery(Duration.ofMillis(10));
+        wait.until(ExpectedConditions.textToBePresentInElement(driver.findElement(loadingPercentage), limit + "%"));
```

Comment:
You verify that text is equals to 50%. But you need to verify that percentage >= 50. Your code works only because of polling frequency every 10 ms. If you change it to 1000 ms, then code will not work. Please reach me out if you have any questions

Reply:
Thanks for clarifying